.env.development.no-cors.sample

```diff
@@ -0,0 +1,3 @@
+FOUNDRY_API_URL=https://<YOUR FOUNDRY STACK DOMAIN>
```

Comment:
i meant something a bit different - if we wanted to keep the old env variables i was suggesting having them in a file called `.env.development.no-cors.sample`. this one (the default) should still be called `.env.development.sample`

Reply:
(so basically swap the names)